docs/docs/native-docker.md

```diff
@@ -280,12 +279,11 @@ the future, as Mesos may not always interact with Docker via the CLI.
 }
 ```
 
-#### Docker container support without Docker Engine
+#### Docker Container Support without Docker Engine
 
-Starting with version 1.3.0, Marathon supports docker container images without having the Docker Containerizer
-depend on a Docker Engine. Instead the Mesos containerizer with native AppC support added in Apache Mesos version 1.0
-(released July 2016) directly uses native OS features to configure and start Docker containers and to provide isolation.
-This setup is selected by specifying a JSON combination that used to provoke an error message:
+Starting with version 1.3.0, Marathon supports Docker container images without having the Docker containerizer
+depend on a Docker engine. Instead, the Mesos containerizer with native [AppC](https://github.com/appc/spec) support (added in [Apache Mesos version 1.0](http://mesos.apache.org/blog/mesos-1-0-0-released/), released July 2016) uses native OS features directly to configure and start Docker containers and provide isolation.
```

Comment:
I think we should rephrase this a bit, AppC support seems a bit too emphasised, whereas it's actually Docker support we want to talk about here. I'd prefer: "Instead, the Mesos containerizer (added in [Apache Mesos version 1.0](http://mesos.apache.org/blog/mesos-1-0-0-released/), released July 2016) uses native OS features directly to configure and start Docker or [AppC](https://github.com/appc/spec) containers and provide isolation."